templates/mautic/0/rancher-compose.yml

```diff
@@ -0,0 +1,41 @@
+.catalog:
+  name: "Mautic"
+  version: "v1.0.0"
+  description: "Mautic provides free and open source marketing automation software available to everyone. Free email marketing software, lead management software and more."
+  uuid: Mautic-0
+  minimum_rancher_version: v0.51.0
+  questions:    
+
+    - variable: public_port
+      description: "public port to access the mautic site"
+      label: "PUBLIC_PORT"
+      required: true
+      default: "80"
+      type: "int"
+
+    - variable: mautic_db_host
+      description: "mautic database host"
+      label: "MAUTIC_DB_HOST"
+      required: true
+      default: "mysql"
+      type: "string"
+
+    - variable: mautic_db_name
+      description: "mautic database name"
+      label: "MAUTIC_DB_NAME"
+      required: true
+      default: "mautic"
+      type: "string"
+
+    - variable: mautic_db_user
```

Comment:
You are not passing `mautic_db_user` to `mauticdb` service as `MYSQL_USER` env variable. Changes here has no effect in the package.